RecentActivity/src/org/sleuthkit/autopsy/recentactivity/RAImageIngestModule.java

```diff
@@ -57,7 +59,12 @@
     public void startUp(IngestJobContext context) throws IngestModuleException {
         this.context = context;
 
-        Extract registry = new ExtractRegistry();
+        Extract registry = null;
+        try {
+            registry = new ExtractRegistry();
+        } catch (FileNotFoundException ex) {
+            Exceptions.printStackTrace(ex);
```

Comment:
This should all go away when we throw only IngestModuleExceptions.  And, we should never just print the stack. We need to log it or act on it.